### ISSUE TITLE
Created models should include part x.serializer.g.dart 

### ISSUE THREAD

**Issue by thosakwe** [angel-dart/cli#80]

**Comment by hackerunet** [comment]
Sorry to reopen this, but, when I use angel make model I got 2 files created besides the model class itself and also the angel_serialize.dart is not included in the default pubsec.yaml created after creating a new project from CLI. so do you have a documentation about what are this 2 files missing and how to proceed with the CLI in order to create full MVC architecture? 
the files created are this:
part 'users.g.dart';
part 'users.serializer.g.dart';
Sorry, the files were not created, the CLI only included both parts calls but no files were created only the model was created.

**Comment by thosakwe** [comment]
I suggest that you join the Angel chat: https://gitter.im/angel_dart/discussion

1. Are you using Angel 2?
2. The default boilerplate does not include angel_serialize, because not every project needs it. Check its docs out: https://github.com/angel-dart/serialize
3. The CLI *used to* install the missing dependency, but when Dart2Stable landed, `package:pubspec` broke, so I had to switch to `package:pubspec_parse`, which doesn't support regenerating pubspecs. At this point, though, both libraries have been updated, so I might eventually be able to bring this functionality back.
